.github/workflows/notify-on-push.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         if: ${{ env.PR_ID }}
         run: |
           gh pr comment ${{ env.PR_ID }} --repo "${{ github.repository }}" \
-            --body "A commit referencing this Pull Request was pushed to `main` by @${{ github.actor }} without the expected commit metadata added to its message."
+            --body "A commit referencing this Pull Request was pushed to `${{ github.ref_name }}` by @${{ github.actor }} with an invalid commit message."
```

Comment:
Non-blocking: Would it be better to say that the commit message failed linting and/or that it is in an invalid format rather than saying "invalid commit message"?